pkg/client/client_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/codeready-toolchain/api/pkg/apis"
+	"github.com/codeready-toolchain/api/api/v1alpha1"
```

Comment:
Could we all agree to use `toolchainv1alpha1` as an alias for `github.com/codeready-toolchain/api/api/v1alpha1` in all our codebase? 
IMHO, `v1alpha1` alone is a bit meaningless. Also, we've been doing something similar for other packages we import. Eg:
```
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Reply:
sure - done in f9d3333

Reply:
awesome, thanks!